legal-api/tests/unit/api/test_nr_type_map.py

```diff
@@ -0,0 +1,94 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests to assure the Name Request Type mapping service.
+
+Test-Suite to ensure that the /nrTypeMap endpoint is working as expected.
+"""
+
+# test with no parameters
+def test_no_parameters(client):
```

Comment:
consider collapsing these to a parameterized aka table test.
makes it easier for the next person to add cases.

Reply:
Parameterized and complete.